CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
-## 3.8.2 (TBC)
+## 3.8.2 (January 12th, 2023)
 
-Changes in progress for v3.8.2 are available at [.changes/v3.8.2](https://github.com/vmware/terraform-provider-vcd/tree/main/.changes/v3.8.2) until the release.
+### IMPROVEMENTS
+* Add `catalog_id` to resource and data source `vcd_catalog_media` to allow handling similarly to `vcd_catalog_vapp_template` ([#972](https://github.com/vmware/terraform-provider-vcd/pull/972))
+
+### BUG FIXES
+* Fix a bug that caused inconsistent plan when using `group_id` in `vcd_catalog_access_control`,
+  `vcd_org_vdc_access_control` and `vcd_vapp_access_control` resources ([#963](https://github.com/vmware/terraform-provider-vcd/pull/963))
+* Change `vcd_catalog`, `vcd_catalog_media`, `vcd_catalog_vapp_template`, and `vcd_catalog_item` to access their entities without the need to use a full Org object, thus allowing the access to shared catalogs from other organizations (Issue #960) ([#972](https://github.com/vmware/terraform-provider-vcd/pull/972))
```

Comment:
I see in the rendered view that `#960` is not linked. Not sure if we want to do it, but just saying.

Reply:
I would also put this bullet at the very top of the list, because it's the main reason for the release.

Reply:
We usually leave Issue numbers without rendering. If we want to start changing that, we need to modify our rendering script.